.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build Area Calculator
+
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+      - "LICENSE"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "LICENSE"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version-file: "pyproject.toml"
+          architecture: "x64"
+          cache: "pip"
+          
+      - name: Install dependencies
+        run: pip install . pyinstaller
```

Comment:
Wouldn't it better to use uv instead? At least we can ensure that the versions are the same as the one in the lock file
https://docs.astral.sh/uv/guides/integration/github/

Reply:
i tried uv method before but pyinstaller wont import packages like numpy, etc.
+ waiting for https://github.com/astral-sh/uv/issues/5802

Reply:
I was able to solve it using dev dependencies
[Run Here](https://github.com/Sclafus/Area_Calculator_Osu/actions/runs/14007885614)